src/routes/createNewHouse.routes.js

```diff
@@ -0,0 +1,14 @@
+import createNewHouseController from '../controllers/createNewHouse.controllers';
```

Comment:
To keep the naming convention/routes storing practices similar to the rest of the code prepared so far, you could consider renaming the file to `houses.routes.js` and store here all house related routes.